Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -195,6 +195,8 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
         let fontNameSection = new WI.DetailsSection("font-identity", WI.UIString("Identity", "Identity @ Font Details Sidebar Section", "Section title for font identity information."), [previewGroup]);
         this.element.appendChild(fontNameSection.element);
 
+        this._fontNameRow.element.style.fontFamily = "font-identity";
```

Comment:
What's the intent of this patch?

`font-identity` is not a valid value for the ["font-family" CSS property](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family). 

Even if it were, this only changes the hint given to the browser about which font to use (if it were successfully loaded with a `@font-face` rule) to style the element from the Font panel in Web Inspector.

Reply:
I aim to display the font identity's name using its corresponding font face/style. Alternatively, I could include a font example ('abc') next to the font name. Would utilizing 'getComputedStyle' be preferable over 'font-identity'?

Reply:
> I aim to display the font identity's name using its corresponding font face/style.

I think I can try to imagine that. A design mock-up of what you intend to build would help. 

Keep in mind that some font files don't have user-friendly names. For example `font-family: system-ui` appears on Mac as `.NS SF`, instead of "San Francisco". Inspect some samples on [v-fonts.com](https://v-fonts.com) and check their names in the Font Identity section of the Font sidebar panel to see what I mean.

> Alternatively, I could include a font example ('abc') next to the font name.

A common pattern like that might be more useful to help distinguish the sample from the font name.

> Would utilizing 'getComputedStyle' be preferable over 'font-identity'?

- `font-identity` it not a known `font-family` value or supported keyword in CSS.
- remember that this code runs in Web Inspector's frontend context. Think of Web Inspector as a website / web app. `getComputedStyle()` in this context will return values from Web Inspector's `window`, not the inspected page's `window`. 

You'll have to get the rendered font samples from the inspected page via the Web Inspector backend. 
I don't _think_ you can/should get the URLs to the font files and pass them over to Web Inspector's frontend. More likely, you'll need to get a rendering (image, base64 encoded) of the sample text styled with the appropriate font from the backend and pass to the frontend to display. (FIWIW, that's how Firefox DevTools does it)

Reply:
Web Inspector is able to preview fonts (more specifically font resources) inside `WI.FontResourceContentView`.  perhaps take a look at that and see if there's any way to reuse some of that for this?

generally speaking i am in favor of the idea of showing a preview of the font (or at least adding something that'd jump to the font resource in the Sources Tab), but this is not the way to do it (i.e. as @rcaliman-apple said `font-identity` is not a real thing and wont do anything, so you'll have to do something else to make this work)